lib/python/rose/config_editor/variable.py

```diff
@@ -186,8 +180,8 @@ def generate_valuewidget(self, variable, override_custom=False,
                                           set_value,
                                           hook_object,
                                           custom_arg)
-            except Exception as e:
-                self.handle_bad_valuewidget(str(e), variable, set_value)
+            except Exception as exc:
```

Comment:
Can this type of error be given a specific exception class instead?

Reply:
A bit more difficult, because `widget` can be anything.